src/GrumPHP/Console/Command/RunCommand.php

```diff
@@ -60,7 +60,7 @@ protected function configure()
     public function execute(InputInterface $input, OutputInterface $output)
     {
         $files = $this->getRegisteredFiles();
-        $context = new RunContext($files);
+        $context = new RunContext($files, $this->paths());
```

Comment:
I don't really want the paths available in any context.
Since it is a console helper I also don't want them in the tasks.
Is there another way?